package.json

```diff
@@ -18,7 +18,7 @@
   },
   "devDependencies": {
     "@babel/core": "^7.12.9",
-    "@types/react": "~18.0.14",
+    "@types/react": "^18.0.18",
```

Comment:
types/react version has changed to 18.0.18
Will package versions be changing every time someone npm installs?